/**:
  ros__parameters:
    gyro_bias_threshold: 0.001 # [rad/s]
    velocity_threshold: 0.0 # [m/s]
    timestamp_threshold: 0.1 # [s]
    data_num_threshold: 1000 # [num]
